From cf4acf84c6304d34108dadd5e87c9a19ca24dceb Mon Sep 17 00:00:00 2001
From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Wed, 1 Oct 2014 23:15:34 +0200
Subject: [PATCH] Remove repeated includes

In pty.c there was both an include of our pty.h and the system installed pty.h.
The latter contains only two functions openpty and forkpty. We use neither so
I assume it was a typo and removed it. We still compile and pass all tests.
---
 src/bus-proxyd/bus-proxyd.c         | 1 -
 src/bus-proxyd/test-bus-policy.c    | 1 -
 src/shared/locale-util.c            | 1 -
 src/shared/missing.h                | 1 -
 src/shared/pty.c                    | 1 -
 src/sysv-generator/sysv-generator.c | 1 -
 6 files changed, 6 deletions(-)

diff --git a/src/bus-proxyd/bus-proxyd.c b/src/bus-proxyd/bus-proxyd.c
index 6a0fc7edfb..5d58cd2af4 100644
--- a/src/bus-proxyd/bus-proxyd.c
+++ b/src/bus-proxyd/bus-proxyd.c
@@ -40,7 +40,6 @@
 #include "bus-internal.h"
 #include "bus-message.h"
 #include "bus-util.h"
-#include "bus-internal.h"
 #include "build.h"
 #include "strv.h"
 #include "def.h"
diff --git a/src/bus-proxyd/test-bus-policy.c b/src/bus-proxyd/test-bus-policy.c
index 7bcebef905..5267788b70 100644
--- a/src/bus-proxyd/test-bus-policy.c
+++ b/src/bus-proxyd/test-bus-policy.c
@@ -36,7 +36,6 @@
 #include "bus-internal.h"
 #include "bus-message.h"
 #include "bus-util.h"
-#include "bus-internal.h"
 #include "build.h"
 #include "strv.h"
 #include "def.h"
diff --git a/src/shared/locale-util.c b/src/shared/locale-util.c
index d5eaff3d9e..9addb05f09 100644
--- a/src/shared/locale-util.c
+++ b/src/shared/locale-util.c
@@ -25,7 +25,6 @@
 #include "util.h"
 #include "utf8.h"
 #include "strv.h"
-#include "util.h"
 
 #include "locale-util.h"
 
diff --git a/src/shared/missing.h b/src/shared/missing.h
index b441149945..031fe2d1ce 100644
--- a/src/shared/missing.h
+++ b/src/shared/missing.h
@@ -33,7 +33,6 @@
 #include <linux/input.h>
 #include <linux/if_link.h>
 #include <linux/loop.h>
-#include <linux/if_link.h>
 
 #ifdef HAVE_AUDIT
 #include <libaudit.h>
diff --git a/src/shared/pty.c b/src/shared/pty.c
index 31123e5af2..adcb32d0be 100644
--- a/src/shared/pty.c
+++ b/src/shared/pty.c
@@ -46,7 +46,6 @@
 #include <fcntl.h>
 #include <limits.h>
 #include <linux/ioctl.h>
-#include <pty.h>
 #include <signal.h>
 #include <stdbool.h>
 #include <stdint.h>
diff --git a/src/sysv-generator/sysv-generator.c b/src/sysv-generator/sysv-generator.c
index 43bcaa862f..6e4cdd694e 100644
--- a/src/sysv-generator/sysv-generator.c
+++ b/src/sysv-generator/sysv-generator.c
@@ -31,7 +31,6 @@
 #include "path-util.h"
 #include "path-lookup.h"
 #include "log.h"
-#include "strv.h"
 #include "unit.h"
 #include "unit-name.h"
 #include "special.h"
